package.json

```diff
@@ -35,28 +35,28 @@
     "debug": "^4.3.3",
     "execa": "^5.1.1",
     "lilconfig": "2.0.4",
-    "listr2": "^3.13.5",
+    "listr2": "^4.0.1",
```

Comment:
No breaking changes relevant to us here?

Reply:
None that I could figure out from the release log:
https://github.com/cenk1cenk2/listr2/releases

Reply:
>This changes the default behavior of error collection mechanism.

I don't think we use this feature, since we collect the task errors separately.

Reply:
Also, Node.js 10 is unsupported already.

Reply:
Let me test a bit locally first...

Reply:
Yeah, seems to work fine 👍